crates/client/src/account/register.rs

```diff
@@ -41,10 +49,56 @@ pub fn register() -> Html {
                 placeholder="Username"
                 name="username"
                 id="username"
-                type="username"
+                type="text"
             />
             <button type="submit">{ "Register" }</button>
         </form>
         </>
     }
 }
+
+fn hash_password(password: &str) -> String {
+    let mut hasher = Sha256::new();
```

Comment:
Sha256 is not secure now for storing passwords, better use Argon2 for better security
https://github.com/ArlekinTeam/ArlekinClient2023/blob/49ac627836bca0db3dd178baa3c3f5be988d9fb7/frontend/src/account/login.rs#L107C1-L113C23

Reply:
I will implement this later